docs/core/docker/publish-as-container.md

```diff
@@ -143,19 +180,50 @@ The container base image property controls the image used as the basis for your
 - If your project is an ASP.NET Core project, the `mcr.microsoft.com/dotnet/aspnet` image is used as the base image.
 - Otherwise the `mcr.microsoft.com/dotnet/runtime` image is used as the base image.
 
-The tag of the image is inferred to be the numeric component of your chosen `TargetFramework`. For example, a project targeting `.net6.0` will result in the `6.0` tag of the inferred base image, and a `.net7.0-linux` project will use the `7.0` tag.
+The tag of the image is inferred to be the numeric component of your chosen `TargetFramework`. For example, a project targeting `net6.0` results in the `6.0` tag of the inferred base image, and a `net7.0-linux` project uses the `7.0` tag, and so on.
 
 If you set a value here, you should set the fully qualified name of the image to use as the base, including any tag you prefer:
 
+:::zone pivot="dotnet-8-0"
+
+```xml
+<PropertyGroup>
+    <ContainerBaseImage>mcr.microsoft.com/dotnet/runtime:8.0</ContainerBaseImage>
+</PropertyGroup>
+```
+
+:::zone-end
+:::zone pivot="dotnet-7-0"
+
 ```xml
 <PropertyGroup>
-    <ContainerBaseImage>mcr.microsoft.com/dotnet/runtime:6.0</ContainerBaseImage>
+    <ContainerBaseImage>mcr.microsoft.com/dotnet/runtime:7.0</ContainerBaseImage>
 </PropertyGroup>
 ```
 
+:::zone-end
+
+:::zone pivot="dotnet-8-0"
+
+### ContainerFamily
+
+Starting with .NET 8, `ContainerFamily` is used to choose a different family of Microsoft-provided container images as the base image for your app. When set, this value is appended to the end of the selected TFM-specific tag, changing the tag provided. For example, to use the Alpine Linux variants of the .NET base images, you can set `ContainerFamily` to `alpine`:
+
+```xml
+<PropertyGroup>
+    <ContainerFamily>alpine</ContainerFamily>
+</PropertyGroup>
+```
+
+The preceding project configuration results in a final tag of `8.0-alpine` for a .NET 8-targeting app.
+
+This field is free-form, and often can be used to select different operating system distributions, default package configurations, or any other _flavor_ of changes to a base image. For more information, see [.NET container images](container-images.md).
```

Comment:
These caveats are great 👍 I'd add one more - if the user sets `ContainerBaseImage`, this field does nothing.